tests/sklearn/test_SklearnGLMClassifierConverter.py

```diff
@@ -62,6 +63,16 @@ def test_model_sgd_multi_class(self):
         self.assertIsNotNone(model_onnx)
         dump_data_and_model(X.astype(numpy.float32), model, model_onnx, basename="SklearnSGDClassifierMulti-Dec3")
 
+    def test_model_knn_classifier_binary_class(self):
+        model, X = self._fit_model_binary_classification(KNeighborsClassifier())
+        model_onnx = convert_sklearn(model, 'KNN classifier', [('input', FloatTensorType([1, 3]))])
+        self.assertIsNotNone(model_onnx)
+
+    def test_model_knn_classifier_multi_class(self):
+        model, X = self._fit_model_multiclass_classification(KNeighborsClassifier())
+        model_onnx = convert_sklearn(model, 'KNN classifier', [('input', FloatTensorType([1, 3]))])
+        self.assertIsNotNone(model_onnx)
+
```

Comment:
Why not checking the runtime?

Reply:
You can add this for the binary classifier if your test works with the onnxruntime release on pypi:

```
dump_data_and_model(X, model, model_onnx, basename="SklearnBinKNeighborsClassifier")
```

Or this one to tell the test if does not work with the current pypi version but it will with the next release:

```
dump_data_and_model(X, model, model_onnx, basename="SklearnBinKNeighborsClassifier",
                    allow_failure="StrictVersion(onnxruntime.__version__) <= StrictVersion('0.1.3')")
```

Just change SklearnBinKNeighborsClassifier by SklearnMclKNeighborsClassifier. The name in itself is not important but it has to be unique for this test. I chose to prefix every file by the name of the library for clarity as every converted model is saved on disk based on this name.

Reply:
Added